back/engines/commercial/impact_tracking/app/controllers/impact_tracking/web_api/v1/pageviews_controller.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+module ImpactTracking
+  module WebApi::V1
+    class PageviewsController < ::ApplicationController
+      skip_before_action :authenticate_user, only: :create
+      skip_after_action :verify_authorized, only: :create
+
+      # POST /sessions/:id/track_pageview
+      def create
+        return head(:not_found) unless Session.exists?(params[:id])
+
+        path = pageview_params[:client_path]
+        route = pageview_params[:route]
+
+        project_id = derive_project_id(path, route)
```

Comment:
As we briefly discussed during our call, in the ideal world, I think this information would be annotated in the front-end routes and passed through to the endpoint. It keeps it nicely centralized and makes it easier to maintain.

But I get that this was probably a lot easier, and will likely cover what we need, so I'm fine with it.